ci/jenkins/jobs/projects-lab.yaml

```diff
@@ -1339,3 +1339,110 @@
               default-excludes: true
               fingerprint: false
               only-if-success: false
+      - '{name}-{test_name}-for-pull-request':
+          test_name: nsxvpc-e2e
+          node: 'antrea-nsxvpc-testbed'
+          description: 'This is the {test_name} test for {name}.'
+          branches:
+          - ${{sha1}}
+          builders:
+            - builder-e2e
+          trigger_phrase: ^(?!Thanks for your PR).*/test-nsxvpc-(e2e|all).*
```

Comment:
Question: The major difference for the vpc and other CI jobs is the target testbed, right?

Reply:
Yes，there is no difference in CI script.